src/sentry/api/helpers/group_index/update.py

```diff
@@ -419,9 +416,19 @@ def update_groups(
                             release_version=release.version,
                         )
 
-                        current_release_version = get_current_release_version_of_group(
```

Comment:
I think we should look to remove this as a follow-up https://github.com/getsentry/sentry/blob/c0797226ec0dc304d766796713121a880df38811/src/sentry/integrations/tasks/sync_status_inbound.py#L112-L114 and I think that actually that code path should have the same fix you're adding in here. So might be worth also adding the fix there as part of this

Reply:
Definitely. I was trying to mostly working and not being awful code to review.

Reply:
Sounds good!

Reply:
I followed-up here: https://github.com/getsentry/sentry/pull/80821